dataflow/src/org/checkerframework/dataflow/analysis/FlowExpressions.java

```diff
@@ -478,7 +480,9 @@ public boolean equals(Object obj) {
                 return false;
             }
             LocalVariable other = (LocalVariable) obj;
-            return other.element.equals(element);
+            return other.element.name.toString().equals(element.name.toString()) &&
+                   other.element.type.toString().equals(element.type.toString()) &&
```

Comment:
Can you add a comment why this uses toString comparison instead of com.sun.tools.javac.code.Types.isSameType(Type, Type)